expan/core/experiment.py

```diff
@@ -144,3 +144,57 @@ def delta(self, method='fixed_horizon', **worker_args):
 
         result['kpis'] = kpis
         return result
+
+    def _quantile_filtering(self, kpis, percentile, kind):
+        flags = pd.Series(data=[False]*len(self.data))
+        for column in self.data[kpis].columns:
+            threshold = np.percentile(self.data[column], percentile)
+            if kind == 'upper':
+                flags = flags | self.data[column].apply(
+                    lambda x: x >= threshold,
+                )
+            elif kind == 'lower':
+                flags = flags | self.data[column].apply(
+                    lambda x: x <= threshold,
+                )
+        return flags
+
+    def filter(self, kpis, percentile=99.0, kind='upper'):
+        """
+        Method that filters out entities whose KPIs exceed the value at a given percentile.
+        If any of the KPIs exceeds its threshold the entity is filtered out.
+
+        Args:
+            kpis (list): list of KPI names
+            percentile (float): percentile considered as threshold
+
```

Comment:
I would document what does ```kind``` mean.
Besides, the word ```kind``` sounds kind of inaccurate. Maybe rename to ```filter_out_type```?

Reply:
good catch, forgot to document that. will rename it as well.